pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -741,34 +737,6 @@ func doActivate(ctx *zedmanagerContext, uuidStr string,
 		// if the VM already active or in restarting/purging state - continue with the doActivate logic
 	}
 
-	// delay this if referencename is not set
-	if ctx.hvTypeKube && config.FixedResources.VirtualizationMode == types.NOHYPER {
-		var findcontainer bool
-		for _, vrc := range config.VolumeRefConfigList {
-			vrs := lookupVolumeRefStatus(ctx, vrc.Key())
-			if vrs == nil || !vrs.IsContainer() {
-				continue
-			}
-			findcontainer = true
-			if vrs.ReferenceName == "" {
-				log.Noticef("doActivate: waiting for referencename ")
-				if status.State != types.START_DELAYED {
-					status.State = types.START_DELAYED
-					return true
-				}
-				return changed
-			}
-		}
-		if !findcontainer {
-			log.Noticef("doActivate: no container found, wait")
-			if status.State != types.START_DELAYED {
-				status.State = types.START_DELAYED
-				return true
-			}
-			return changed
-		}
-	}
-
```

Comment:
have we tried the native container failover testing w/ this PR? just want to make sure this does not cause the reference-name issue.

Reply:
Yes I tested and it fails as you mentioned. But we do not support native containers yet in product or ST testing and want to get this to systest ASAP. Will fix the native containers issue in next builds.